packages/@aws-cdk-testing/framework-integ/test/aws-eks/test/integ.eks-subnet-updates.ts

```diff
@@ -0,0 +1,32 @@
+import { App, Stack } from 'aws-cdk-lib';
+import * as eks from 'aws-cdk-lib/aws-eks';
+import * as ec2 from 'aws-cdk-lib/aws-ec2';
+import * as integ from '@aws-cdk/integ-tests-alpha';
+import { getClusterVersionConfig } from './integ-tests-kubernetes-version';
+
+class EksClusterStack extends Stack {
+
+  constructor(scope: App, id: string) {
+    super(scope, id);
+
+    const vpc = new ec2.Vpc(this, 'Vpc', { natGateways: 1 });
+    new eks.Cluster(this, 'Cluster', {
+      vpc,
+      ...getClusterVersionConfig(this, eks.KubernetesVersion.V1_29),
+      defaultCapacity: 0,
+      endpointAccess: eks.EndpointAccess.PUBLIC_AND_PRIVATE,
+      vpcSubnets: [{ subnetType: ec2.SubnetType.PUBLIC }],
+    });
+  }
+}
+
+const app = new App();
+
+const stack = new EksClusterStack(app, 'aws-cdk-eks-cluster-stack');
```

Comment:
How does this test test subnet updates? It only get deployed once right?

Reply:
Yes the flow was manually tested, and then changed to this integ test here.